app/src/main/java/com/yeonae/chamelezone/view/map/MapTabFragment.kt

```diff
@@ -57,7 +57,7 @@ class MapTabFragment : Fragment(), OnMapReadyCallback, MapContract.View,
         if (marker != null) {
             (activity as? HomeActivity)?.replace(
                 SingleInfoFragment.newInstance(
-                    marker.title
+                    "${edt_search.text}".replace(" ", "")
```

Comment:
```suggestion
                    "${edt_search.text}".trim()
```
공백 제거를 위함이라면 이런 함수가 있습니다.

Reply:
수정했습니당
f6fd53b